.github/workflows/publish.yml

```diff
@@ -9,27 +9,16 @@ jobs:
   github-pages:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
-          working-directory: legacy
-      - uses: actions/cache@v1
-        with:
-          path: legacy/vendor/bundle
-          key: ${{ runner.os }}-gems-v2-${{ hashFiles('**/Gemfile.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-gems-v2
-      - run: gem install bundler
+      - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 18
       - run: yarn install --frozen-lock-file
       - run: yarn lint
       - run: yarn test
-      - run: ./build.sh --mode production
+      - run: ./build.sh
       - uses: JamesIves/github-pages-deploy-action@3.7.1
         with:
           GITHUB_TOKEN: ${{ secrets.JEKYLL_PAT }}
```

Comment:
@AndyClifford Suggestion: Remove all references to Jekyll

Reply:
Can you please duplicate the value of `JEKYLL_PAT` to `GITHUB_TOKEN`